config/settings.yml

```diff
@@ -394,7 +394,11 @@ reports:
 oidc:
   auth_server_metadata_url: ~
   issuer: ~
+  issuer_prefix: ~
   audience: ~
```

Comment:
At the time, I was concerned about how best to do environment by environment configurations on the devops yaml side, so used the existing `audience` setting, since there isn't a DU-like structure with vets-api to promote carefully environment by environment.  Figured it would minimize risk by using the existing configs by default, and override only if the `isolated_audience` was present.

I am fine with either `isolated_audience` or `audiences` if you have a preference.

Reply:
Cool, I'm good with no changes.